.cbmc-batch/jobs/aws_cryptosdk_edk_list_init/README.md

```diff
@@ -0,0 +1,10 @@
+# Expected Coverage 
+
+0.97 (62 lines out of 64 statically-reachable lines in 13 functions reached)
+0.97 (62 lines out of 64 statically-reachable lines in 13 statically-reachable functions)
+
+## Expected Functions with Incomplete Coverage 
+
+(10/11)	aws_array_list_is_valid: list is never NULL
+
+(4/5) aws_mul_u64_checked: Multiplication never results in an overflow. 
```

Comment:
Could you follow the same style we used for [MQTT proofs](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v4_beta/cbmc/proofs/IotMqtt_PublishAsync)?

Reply:
Yes! I've gone ahead and made these changes everywhere.